Network/DNS/Lookup.hs

```diff
@@ -429,3 +430,26 @@ lookupSRV rlv dom = do
+
+lookupAXFR :: Resolver -> Domain -> IO (Either DNSError [ResourceRecord])
+lookupAXFR rlv dom = do
+  erds <- DNS.lookupRaw rlv dom AXFR
```

Comment:
I think that `AXFR` should use its own low-level "lookup" interface, rather than lookupRaw, and should probably return a stream, rather than a list of records.  The zones I AXFR have millions of records, and should not be stored as a single array in memory.

Reply:
Makes sense